src/components/CallBooking/CallBooking.tsx

```diff
@@ -17,21 +19,17 @@ const EmptyState = ({ onBookCall }: { onBookCall?: () => void }) => (
       Book a call with your bookkeeper
     </Span>
     <VStack gap='xs'>
-      <Button variant={ButtonVariant.primary} onClick={onBookCall}>Book a call</Button>
+      <Button variant='solid' onClick={onBookCall}>Book a call</Button>
     </VStack>
   </VStack>
 )
 
 const ScheduledCallState = ({
   callBooking,
-  onAddToCalendar: onAdd,
 }: {
   callBooking: CallBookingData
-  onAddToCalendar: (callBooking: CallBookingData) => void
 }) => {
-  const onAddToCalendar = useCallback(() => {
-    onAdd(callBooking)
-  }, [callBooking, onAdd])
+  const purpose = callBooking.purpose === CallBookingPurpose.BOOKKEEPING_ONBOARDING ? 'Onboarding call' : 'Adhoc call'
```

Comment:
NIT: Ad hoc is two words
```suggestion
  const purpose = callBooking.purpose === CallBookingPurpose.BOOKKEEPING_ONBOARDING ? 'Onboarding call' : 'Ad hoc call'
```

Reply:
TIL! Thank you